### ISSUE TITLE
Error when using both "-m 10000" and "-i 3"

### ISSUE THREAD

**Issue by sjfleck** [fenderglass/ABruijn#406]
Thank you for this tool. I'm working with a plant genome projected to be 930 MB. Unfortunately, I only have about 17x coverage (I have 20GB of ONT reads with a read N50 of 34KB and flye is creating a 1.15GB assembly, which is within expectations). I know 17x coverage isn't great and Flye would really like 40x. That being said, while the assemblies aren't especially contiguous (17,000 contigs), the BUSCO scores are pretty phenomenal for a draft assembly (95+% complete).

I'm trying to tweak different parameters to make a better assembly (we are planing on producing more reads, but we want to see what we can do with what we have right now). I've successfully run flye using these options:
1. default settings
2. --polish-target on the default assembly
3. -i 3
4. -m 10000

But for some reason, when I run flye with the "-m 10000" and "-i 3" together, I get an error:

[2021-07-26 19:14:16] INFO: Polishing genome (3/3)
[2021-07-26 19:14:54] INFO: Running minimap2
[2021-07-26 20:21:58] INFO: Separating alignment into bubbles
Traceback (most recent call last):
  File "/projects/academic/vaalbert/modulefiles/anaconda3/envs/flye-2.8.3/bin/flye", line 33, in <module>
    sys.exit(load_entry_point('flye==2.8.3', 'console_scripts', 'flye')())
  File "/projects/academic/vaalbert/modulefiles/anaconda3/envs/flye-2.8.3/lib/python3.9/site-packages/flye/main.py", line 805, in main
    _run(args)
  File "/projects/academic/vaalbert/modulefiles/anaconda3/envs/flye-2.8.3/lib/python3.9/site-packages/flye/main.py", line 584, in _run
    jobs[i].run()
  File "/projects/academic/vaalbert/modulefiles/anaconda3/envs/flye-2.8.3/lib/python3.9/site-packages/flye/main.py", line 366, in run
    pol.polish(self.in_contigs, self.args.reads, self.polishing_dir,
  File "/projects/academic/vaalbert/modulefiles/anaconda3/envs/flye-2.8.3/lib/python3.9/site-packages/flye/polishing/polish.py", line 93, in polish
    make_bubbles(alignment_file, contigs_info, chunks_file,
  File "/projects/academic/vaalbert/modulefiles/anaconda3/envs/flye-2.8.3/lib/python3.9/site-packages/flye/polishing/bubbles.py", line 117, in make_bubbles
    t.start()
  File "/projects/academic/vaalbert/modulefiles/anaconda3/envs/flye-2.8.3/lib/python3.9/multiprocessing/process.py", line 121, in start
    self._popen = self._Popen(self)
  File "/projects/academic/vaalbert/modulefiles/anaconda3/envs/flye-2.8.3/lib/python3.9/multiprocessing/context.py", line 224, in _Popen
    return _default_context.get_context().Process._Popen(process_obj)
  File "/projects/academic/vaalbert/modulefiles/anaconda3/envs/flye-2.8.3/lib/python3.9/multiprocessing/context.py", line 277, in _Popen
    return Popen(process_obj)
  File "/projects/academic/vaalbert/modulefiles/anaconda3/envs/flye-2.8.3/lib/python3.9/multiprocessing/popen_fork.py", line 19, in __init__
    self._launch(process_obj)
  File "/projects/academic/vaalbert/modulefiles/anaconda3/envs/flye-2.8.3/lib/python3.9/multiprocessing/popen_fork.py", line 66, in _launch
    self.pid = os.fork()
OSError: [Errno 12] Cannot allocate memory
Process Process-72:
[2021-07-26 20:22:46] ERROR: Thread exception
... and so on

I also deleted the "40-polishing" directory and tried the resume option and it had the exact same error when it got to the 3rd polishing step. 

Any help you can provide for why I'm getting this error now would be greatly appreciated. Let me know if you need to see more of the error file and thanks in advance!

**Comment by mikolmogorov** [comment]
The log says that system ran out of memory - not sure why. I doubt that 3 polishing iterations would make a difference though. I would stick to 1 polishing iteration + Medaka. If you still want to do 3 iterations, you might try the latest source code (not a part of a release yet) which has some updates.

**Comment by sjfleck** [comment]
I'm not reopening this issue, but the problem was solved by running Flye-2.8.3 on a node with higher memory. Previously, I ran it on a node with 187GB of memory and it failed (even though it reported to use <100GB of memory overall). I switched it to a node with 256GB of memory and it worked. Both nodes could use 32 threads, so the memory/thread must have exceeded some limit in the run with 187GB mem.

I had this same issue with this species when I reran it with Flye-2.9. Funny enough, it finished without error when I ran it with the "-i 3" and "--scaffold" options, but it had node failures when I ran it with "-i 3" alone. Again, running it on a node with higher memory fixed the issue.
